src/services/authorization.ts

```diff
@@ -305,6 +309,7 @@ export const filterOutPackedDescendants = async (
   repositories,
```

Comment:
I know it's not yours but I'm triggered by this one
```suggestion
  repositories: Repositories,
```